simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/ReaderActivity.kt

```diff
@@ -149,6 +164,19 @@ class ReaderActivity : AppCompatActivity(), SR2ControllerHostType {
         .beginTransaction()
         .replace(R.id.reader_container, readerFragment)
         .commit()
+
+      this.analyticsService.publishEvent(
+        AnalyticsEvent.BookOpened(
+          timestamp = LocalDateTime.now(),
+          credentials = this.account.loginState.credentials,
+          profileUUID = this.currentProfile.id.uuid,
+          profileDisplayName = this.currentProfile.displayName,
+          accountProvider = this.account.provider.id,
+          accountUUID = this.account.id.uuid,
+          opdsEntry = this.bookEntry.feedEntry,
+          targetURI = this.bookEntry.feedEntry.analytics.getOrNull()
```

Comment:
I'm pretty sure this is the right value to set here but I saw some funky results for some books. @io7m any ideas?